### PR TITLE
SSCS-10895 remove PO attending from override fields

### PR DESCRIPTION
**Pull request by nickhill111**
SSCS-10895

**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

https://tools.hmcts.net/jira/browse/SSCS-10895

Updated hearings API to allow for the commons changes of removing poToAttend from the override fields

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X ] No
```

### CODE REVIEW

**Review by bensonlievoco — DISMISSED** [review]

**Review by TeaWithLucas — APPROVED** [review]

**Review by bensonlievoco — APPROVED** [review]

**Review by TeaWithLucas — APPROVED** [review]

**Review by dattipoeHMCTS — APPROVED** [review]

**Review by dattipoeHMCTS — APPROVED** [review]
Don;t forget the common release

**Review by dattipoeHMCTS — APPROVED** [review]

**Review by EliandroGV1 — DISMISSED** [review]

**Review by EliandroGV1 — APPROVED** [review]

File: src/test/java/uk/gov/hmcts/reform/sscs/service/HearingsServiceTest.java [hunk 1]
```diff
@@ -62,12 +62,7 @@
 @ExtendWith(MockitoExtension.class)
 class HearingsServiceTest {
     private static final long HEARING_REQUEST_ID = 12345;
-    private static final String HMC_STATUS = "TestStatus";
```
Comment (hgulv1) [inline src/test/java/uk/gov/hmcts/reform/sscs/service/HearingsServiceTest.java:65]: were these removed as part of a refactor?

File: src/test/java/uk/gov/hmcts/reform/sscs/service/HearingsServiceTest.java [hunk 2]
```diff
@@ -62,12 +62,7 @@
...
-    private static final long VERSION = 1;
```
Comment (hgulv1) [inline src/test/java/uk/gov/hmcts/reform/sscs/service/HearingsServiceTest.java:66]: same as above

File: src/test/java/uk/gov/hmcts/reform/sscs/service/HearingsServiceTest.java [hunk 3]
```diff
@@ -62,12 +62,7 @@
...
     private static final long CASE_ID = 1625080769409918L;
-    private static final long MISSING_CASE_ID = 99250807409918L;
-    private static final String IDAM_OAUTH2_TOKEN = "TestOauth2Token";
-    private static final String SERVICE_AUTHORIZATION = "TestServiceAuthorization";
```
Comment (hgulv1) [inline src/test/java/uk/gov/hmcts/reform/sscs/service/HearingsServiceTest.java:70]: same as above

**Review by hgulv1 — CHANGES_REQUESTED** [review]
see comments

File: src/test/java/uk/gov/hmcts/reform/sscs/service/HearingsServiceTest.java (hunk 1 above)
Comment (nickhill111) [reply src/test/java/uk/gov/hmcts/reform/sscs/service/HearingsServiceTest.java:65]: They were unused variables, same with others

**Review by hgulv1 — APPROVED** [review]

**Review by TeaWithLucas — APPROVED** [review]

**Pull request merged**